cmake/CMakeLists.txt

```diff
@@ -960,33 +960,53 @@ if(PKG_KOKKOS)
   endif()
 endif()
 if(PKG_KSPACE)
-  message(STATUS "<<< FFT settings >>>
--- Primary FFT lib:  ${FFT}")
-  if(FFT_SINGLE)
-    message(STATUS "Using single precision FFTs")
-  else()
-    message(STATUS "Using double precision FFTs")
-  endif()
-  if(FFT_FFTW_THREADS OR FFT_MKL_THREADS)
-    message(STATUS "Using threaded FFTs")
+  if (LMP_HEFFTE)
+    message(STATUS "<<< FFT settings >>>
+-- Primary FFT lib:  heFFTe")
+    if (HEFFTE_BACKEND)
+      message(STATUS "heFFTe backend:  ${HEFFTE_BACKEND}")
+    else()
+      message(STATUS "heFFTe backend:  stock (not intended for production)")
```

Comment:
What does this message mean?

Reply:
The `stock` backend is what heFFTe calls the builtin FFT implementation. It's very useful for testing, since we have checked it for correctness and we can use it to verify that interfaces to the other backends work correctly. Similarly, our users can utilize the `stock` backend and check their interface to heFFTe without bothering to install third party libraries, such as FFTW or cuFFT.

However, we don't optimize the `stock` backend for performance across relevant platforms. It behaves well but won't match FFTW or MKL.

I have updated the documentation.